app/models/spree/payment_method/stripe_credit_card.rb

```diff
@@ -81,6 +83,19 @@ def void(response_code, _creditcard, _transaction_options)
         gateway.void(response_code, {})
       end
 
+      def try_void(payment)
+        if payment.completed?
+          reason = Spree::RefundReason.find_by(name: self.class::PAYMENT_INTENTS_REFUND_REASON) || Spree::RefundReason.first
```

Comment:
I think is just necessary `PAYMENTS_INTENTS_REFUND_REASON`, since the constant is in the same class.

Also, instead to fallback to first reason maybe we can use `find_or_create`?

Reply:
@ccarruitero thanks for spotting the issue with the constant, what was  I thinking when I wrote that?

I thought about using `find_or_create`, but then I went for the migration (the record is explicitly created in a more "expected" place) and the fallback to `Spree::RefundReason.first` for making the custom refund reason optional. 
But I think that your options are good as well. I'm wondering if a rake task for adding the refund reason might be a better option... does anybody want to share opinions on this?

Reply:
I have some concerns about creating a migration for this RefundReason. Can't we use an already existing one as default, like `Spree::Payment::Cancellation::DEFAULT_REASON`? Maybe also defined in a separate method so that people can customize it and use something else if needed?